
<p>Python works line by line. The moment python hits an error they will stop execution.</p>

<h2>Example</h2>
<p> a = 5 <br> b = 5 / 0 <br> c = 10</p>
<p>so, we write try and exception handling process what they wanna do when ever we hit an error we send the error some other place and do compleate the task using exception handling.</p>

<h1><center>Process</center></h1>

<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Function will start with the try section what ever you will write in a try section they will start executing one after another 
  <br><br> the moment..
  <br>the moment...
  <br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;They get an error in a try section they will jump to except</p>
<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Then, at the end they jump to Finally.</p>

<p><h1>OR</h1></p>

<p>Suppose they do not find any error in a try part they will never hit except session.They will directly move to Finally.</p>
